doc/sphinx/source/project/release-notes.rst

```diff
@@ -1,41 +1,61 @@
-.. release-notes:
+.. ngraph/release-notes:
 
 Release Notes
 #############
 
-This is |release|.
+The latest |version| download below.
 
+* `Format .zip`_ 
+* `Format tar.gz`_ 
 
+See also: https://github.com/NervanaSystems/ngraph/releases for previous versions. 
 
 
+CHANGELOG |release|
+-------------------
 
-CHANGELOG
-=========
+Release Candidate 0 of v0.17
```

Comment:
How about just `nGraph v0.17.0-rc.0` here?

Reply:
Also, I think we should keep the `.0` after `v0.17.0`. Point releases can have their own `rc`s, e.g. `v0.17.1-rc.3` can be a thing.

Reply:
Will do.  I was trying to take cues from how they are named on our Github releases landpage.

Reply:
Why not `v0.17` so we don't need to keep updating it every candidate? The RCs are all hoping to be the release.

Reply:
That’d be fine with me, except it should be `v0.17.0`. If there are new bug fixes in a point release those should be called out separately.

Reply:
Agree.